boot/core/src/boot/task/built_in.clj

```diff
@@ -493,7 +493,8 @@
    s scm KEY=VAL           {kw str}    "The project scm map (KEY is one of url, tag, connection, developerConnection)."
    l license NAME:URL      {str str}   "The map {name url} of project licenses."
    o developers NAME:EMAIL {str str}   "The map {name email} of project developers."
-   D dependencies SYM:VER  [[sym str]] "The project dependencies vector (overrides boot env dependencies)."]
+   D dependencies SYM:VER  [[sym str]] "The project dependencies vector (overrides boot env dependencies)."
+   a parent SYM:VER        {sym str}   "The project dependency vector of the parent project"]
```

Comment:
@micha I wanted to have `a parent SYM:VER=PATH       [sym str str]` like in #578, can I do it?